crates/runtime/src/embeddings/vector_search.rs

```diff
@@ -721,17 +731,15 @@ impl VectorSearch {
             )
             SELECT
                 substring(t.{embed_col}, rd.offset[1], rd.offset[2] - rd.offset[1]) AS {embed_col}_chunk,
-                {projection_str},
+                {projection_str}
```

Comment:
Are you sure you meant to remove this trailing comma?

Reply:
The comma is accounted for in https://github.com/spiceai/spiceai/pull/4802/files#diff-2099bcaabb894d8accf6f48632574ffa7633d2c2b9c4d56383a84b7e1e8c7a4fR702

The SQL in vector search is likely getting complicated enough to use an unparser.